### ISSUE TITLE
Миксы при большой вложенности

### ISSUE THREAD

**Issue by oficrejosef** [bem-site/bem-forum-content-ru#1368]
Добрый вечер. Такой вопрос, имеется большой блок, скажем, это "block". У поста много всякого и вот есть такой элемент, как лайки и дислайки. Структура упрощенно такая
```
<div class="block">
.....
<div class="block__rating rating rating_type_like">
	<i class="rating__icon"></i>
	Лайк
	<span class="rating__count rating__count_type_like"></span>
</div>
<div class="block__rate rating rating_type_dislike">
	<i class="rating__icon"></i>
	Дислайк
	<span class="rating__count rating__count_type_dislike"></span>
</div>
.......
</div>
```
Вот такое я наваял. 
Как я все понимаю - `block__rating` - собственно показывает, что это элемент блока. Сюда же ввел новый блок  - `rating`, чтобы от него уже получить элементы иконки рейтинга и счетчик.
Изначально у меня все было элементами `block`, то есть `block__icon` и `block__count`, к примеру, но это даже чисто с точки зрения логики как-то неправильно.
Мой вопрос, все ли верно у меня получилось по bem или где-то структурно я не прав?

**Comment by DjonyBastone** [comment]
лайк и дислайк планируется использовать вне блока `block`?

- если будут использоваться вне - то отлично
- если не планируется - то излишне, можно всё сделать элементами блока `block`

П.С. я также мудрил, потом папка уровня переопределения просто была огромной (Словарный запас по англ. почти исчерпал), потом весь проект перетряхивал, что бы уменьшить количество блоков.

**Comment by oficrejosef** [comment]
Нет, использовать вне блока не планирую(по крайней мере сейчас), поэтому и задался вопросом целесообразности. С одной стороны - лишняя сущность, с другой - чисто логически счетчик и иконки это скорее элементы лайков и дизлайков, нежели самого блока..

**Comment by tadatuta** [comment]
@oficrejosef структура норм, если дополнительные сущности не мешают на файловой системе, то нет повода переживать

**Comment by zxqfox** [comment]
Разделять сложнее, чем смешивать.
Если не мешают — лучше оставь так, слить в одно всегда успеешь.
